spec/features/tests/index_view_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
```

Comment:
The name of this spec file should be more descriptive. Can you call it user_views_all_food_trucks_spec.rb or something along those lines? The idea is that scanning through 50 files, you could tell which one pertains to whatever functionality you want to learn about.